modules/ddl-and-loading/pages/add-token-function.adoc

```diff
@@ -173,7 +173,18 @@ After the parameters are successfully configured, you can access your user-defin
 
 == Store token functions locally
 
-=== Step 1: Modify the current TokenBank file
+=== Step 1: Enable adding a TokenBank with PUT
+
+Run the following commands to enable uploading a TokenBank through the GSQL `PUT` command:
+
+[source.wrap,console]
+----
+$ gadmin config set GSQL.UDF.EnableTokenBank true
```

Comment:
The config name is `GSQL.UDF.EnablePutTokenBank`. Sorry for the type in the ticket's description.